index.php

```diff
@@ -79,7 +79,8 @@
         <div class="span12">
           <div class="well">
             <h4>Server Users</h4>
-            <div>
+          <div>
```

Comment:
Sorry, it was a problem






Sent from Windows Mail





From: Gjum
Sent: ‎Saturday‎, ‎January‎ ‎2‎, ‎2016 ‎10‎:‎10‎ ‎PM
To: caseypugh/minecraft
Cc: A T A T





In index.php:
> @@ -79,7 +79,8 @@
>          <div class="span12">
>            <div class="well">
>              <h4>Server Users</h4>
> -            <div>
> +          <div>


There is still no slash here. ;)

—
Reply to this email directly or view it on GitHub.